Sara/index.js

```diff
@@ -27,4 +27,17 @@ let mangoes = 5 ;
 console.log(mangoes * apples > mangoes + oranges ? 'true' : 'false') ;
 
 // 7
-console.log(mangoes - apples < oranges / mangoes ? 'true' : 'false') ; 
\ No newline at end of file
+console.log(apples - mangoes < oranges / mangoes ? 'true' : 'false') ; 
+
+// 8 
+console.log(apples === oranges === mangoes ? 'true' : 'false') ; 
+
+// 9
```

Comment:
Your solution for 9th task is wrong. I send you the correct one:)
console.log((apples % mangoes) == (oranges % mangoes));